lib/danger/comment_generators/github_inline.md.erb

```diff
@@ -0,0 +1,22 @@
+<%- @tables.each do |table| -%>
+  <%- if table[:content].any? -%>
+<table>
+  <tbody>
+    <%- table[:content].each do |violation| -%>
+    <tr>
+      <td>:<%= table[:emoji] %>:</td>
+      <td data-sticky="<%= violation.sticky %>"><%= "<del>" if table[:resolved] %><%= violation.message %><%= "</del>" if table[:resolved] %></td>
+    </tr>
+    <%- end -%>
+  </tbody>
+</table>
+  <%- end -%>
+<%- end -%>
+
+<%- @markdowns.each do |current| -%>
+<%= current %>
+<%# the previous line has to be aligned far to the left, otherwise markdown can break easily %>
+<%- end -%>
+<p align="right" data-meta="generated_by_<%= @danger_id %>">
+  Generated by :no_entry_sign: <a href="http://danger.systems/">danger</a>
+</p>
```

Comment:
I wonder if this may be more annoying, I think we can live without the Danger advert for inline comments 👍